easybuild/easyconfigs/n/ncurses/ncurses-5.9-intel-2015a.eb

```diff
@@ -12,4 +12,7 @@ toolchainopts = {'optarch': True, 'pic': True}
 source_urls = [GNU_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
+# enable UTF-8 version of ncurses
+configopts = '--enable-widec '
```

Comment:
OK, then it makes sense to tackle it here for this ncurses easyconfig, and sync up the others in a dedicated PR.